### PR TITLE
remove rdb subcommand

### PR DESCRIPTION
**Pull request by hanasuke**
I removed `ecl rdb` subcommands.

According to [service descriptions](https://ecl.ntt.com/documents/service-descriptions/), ECL2.0 doesn't have RDB menu.

before (v3.0.0):
```
$ ecl --help rdb
Command "rdb" matches:
  rdb database create
  rdb database delete
  rdb database list
  rdb datastore list
  rdb flavor list
  rdb instance create
  rdb instance delete
  rdb instance list
  rdb instance show
  rdb user create
  rdb user delete
  rdb user grant
  rdb user list
  rdb user revoke
  rdb user show
```

after(this patch):
```
$ ecl --help rdb       
Unknown command ['rdb']
```

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -423,21 +423,6 @@ ecl.vnf.v1 =
     vna_operation_list = eclcli.vnf.v1.operation:ListOperation
     vna_operation_show = eclcli.vnf.v1.operation:ShowOperation
 ecl.database.v1 =
```
Comment (t-wata) [inline setup.cfg]: Please delete this line as well.
```suggestion

```

**Review by t-wata — CHANGES_REQUESTED** [review]
Thanks for your PR.
I have commented on the line that should be removed.

I would also like to ask you to remove the following functions as well, because they are only used in “database" and are not very reusable.

https://github.com/nttcom/eclcli/blob/master/eclcli/common/utils.py#L93-L94
https://github.com/nttcom/eclcli/blob/master/eclcli/common/utils.py#L472-L493

**Comment by hanasuke** [comment]
Thank you for your review!
I removed reviewed points additionally.

**Review by naoshark — APPROVED** [review]

**Review by t-wata — APPROVED** [review]
LGTM 👍 Thank you for your contribution!

**Pull request merged**